doc/source/ray-core/scheduling/placement-group.rst

```diff
@@ -3,58 +3,59 @@ Placement Groups
 
 .. _ray-placement-group-doc-ref:
 
-Placement groups allow users to atomically reserve groups of resources across multiple nodes (i.e., gang scheduling). They can be then used to schedule Ray tasks and actors packed as close as possible for locality (PACK), or spread apart (SPREAD). Placement groups are generally used for gang-scheduling actors, but also support tasks.
+Placement groups allow users to atomically reserve groups of resources across multiple nodes (i.e., gang scheduling).
+They can be then used to schedule Ray tasks and actors packed as close as possible for locality (PACK), or spread apart 
+(SPREAD). Placement groups are generally used for gang-scheduling actors, but also support tasks.
 
-Java demo code in this documentation can be found here `<https://github.com/ray-project/ray/blob/master/java/test/src/main/java/io/ray/docdemo/PlacementGroupDemo.java>`__.
+Here are some real-world use cases:
 
-Here are some use cases:
-
-- **Gang Scheduling**: Your application requires all tasks/actors to be scheduled and start at the same time.
-- **Maximizing data locality**: You'd like to place or schedule your actors close to your data to avoid object transfer overheads.
+- **Distributed Machine Learning Training**: Distributed Training (e.g., :ref:`Ray Train <train-docs>` and :ref:`Ray Tune <tune-main>`) uses the placement group APIs to enable gang scheduling. In these settings, all resources for a trial must be available at the same time. Gang scheduling is a critical technique to enable all-or-nothing scheduling for deep learning training. 
+- **Fault tolerance in distributed training**: Placement groups can be used to configure fault tolerance. In Ray Tune, it can be beneficial to pack related resources from a single trial together, so that a node failure impacts a low number of trials. In libraries that support elastic training (e.g. XGBoost-Ray), spreading the resources across multiple nodes can help to ensure training can be continued even when a node dies.
```

Comment:
```suggestion
- **Fault tolerance in distributed training**: Placement groups can be used to configure fault tolerance. In Ray Tune, it can be beneficial to pack related resources from a single trial together, so that a node failure impacts a low number of trials. In libraries that support elastic training (e.g., XGBoost-Ray), spreading the resources across multiple nodes can help to ensure that training continues even when a node dies.
```

Reply:
Is the term "all-or-nothing scheduling" a known term for this audience? If not, a brief definition and reason to use it would be helpful. I'm new to the term, but something like, "Gang scheduling is a technique where multiple related tasks or processes are scheduled to run simultaneously on a group of nodes to improve system throughput by maximizing the utilization of available resources and reducing communication overheads. Gang scheduling needs to schedule all nodes, otherwise it is not effective."

Reply:
Hmm IIUC  this is not the purpose of gang scheduling for distributed training. @krfricke can you tell me let the purpose of gang scheduling in distributed training? Is it to have up-to-dates weights for all of training workers or sth like that?